src/frontend-pwa/src/components/Button/Button.tsx

```diff
@@ -20,6 +22,12 @@ export function Button({
       size={size}
```

Comment:
I recommend using a default value for props such as size and disabled

Reply:
To clarify, you mean make the optional and if nothing is passed as a prop to size or disabled set a default value?

Reply:
done!